.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+    "githubPullRequests.ignoredPullRequestBranches": [
+        "master"
+    ]
+}
```

Comment:
What does this VS Code workspace setting accomplish?

Reply:
I have no idea. I don't think that is something I have set (at least knowingly). I got a new laptop on Thursday and all settings are still messed up for everything.